lib/hexpm/repository/packages.ex

```diff
@@ -135,4 +135,21 @@ defmodule Hexpm.Repository.Packages do
     |> Repo.all()
     |> Enum.into(%{})
   end
+
+  def accessible_user_owned_packages(user, for_user) do
+    organizations = Users.all_organizations(for_user)
+    organization_ids = Enum.map(organizations, & &1.id)
+
+    # Atoms sort before strings
+    sorter = fn
+               %Organization{name: "hexpm"} ->
+                 :first
+               %Organization{name: name} ->
+                 name
+             end
```

Comment:
We can change this to `fn org -> if(org.name == "hexpm", do: first, else: org.name) end` to keep it shorter.